spec/acceptance/enable_new_tasking_system_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'spec_helper_acceptance'
+
+describe 'initial configuration with older rq worker tasking system' do
```

Comment:
rspec guidelines say you should only have a single top level `describe` while this file has 2. Maybe you also need to add `order: :defined` so it really runs in the order you expect.